.github/workflows/php.yml

```diff
@@ -50,4 +50,5 @@ jobs:
       run: vendor/bin/phpcs --standard=Magento2 Magento2/Helpers/ Magento2/Sniffs
 
     - name: Run ESLint
-        run: ../node_modules/.bin/eslint -c eslint/.eslintrc --rulesdir eslint/rules eslint/rules
+      uses: stefanoeb/eslint-action@1.0.2
```

Comment:
![image](https://user-images.githubusercontent.com/1873745/133281559-e22c848f-b571-4bb3-a05a-e1c9361e419d.png)
looks like that's not the right choise

Reply:
And one more - it doesn't make any sense running it on different PHP versions

Reply:
Not sure if I need the line that is referring to `yarn` since it's something external to the project.
Or maybe `npm install` because we are actually using node_modules...
Anyway, I'm going to try to see if any of these work. Thanks @ihor-sviziev